cli/src/pcluster/utils.py

```diff
@@ -118,31 +118,45 @@ def get_supported_os_for_architecture(architecture):
     return SUPPORTED_OSES_FOR_ARCHITECTURE.get(architecture, [])
 
 
-def camelcase(snake_case_word):
-    """Convert the given snake case word into a camel case one."""
-    parts = iter(snake_case_word.split("_"))
-    return "".join(word.title() for word in parts)
-
-
 def to_iso_time(time_in):
     """
-    Convert a given string or datetime into ISO 8601 format with milliseconds.
+    Convert a given string, datetime or int into ISO 8601 format with milliseconds.
 
     :param time_in: Time in a format that may be parsed
     :return time in ISO 8601 UTC format with ms (e.g. 2021-07-15T01:22:02.655Z)
     """
     if time_in:
-        if isinstance(time_in, str):
+        if isinstance(time_in, int):
+            if time_in > 1e12:
+                time_in /= 1000
+            time_ = datetime.datetime.fromtimestamp(time_in)
+            time_ = time_.replace(tzinfo=datetime.timezone.utc)
+        elif isinstance(time_in, str):
             time_ = dateutil.parser.parse(time_in)
         elif isinstance(time_in, datetime.date):
-            time_ = time_in.replace(tzinfo=datetime.timezone.utc)
+            time_ = time_in
+            if time_.tzinfo is None:
+                time_ = time_.replace(tzinfo=datetime.timezone.utc)
         else:
-            raise TypeError("to_iso_time object must be 'str' or 'datetime'.")
+            raise TypeError("to_iso_time object must be 'str', 'int' or 'datetime'.")
         return time_.isoformat(timespec="milliseconds")[:-6] + "Z"
     else:
         return time_in
 
 
+def to_camel_case(snake_case_word):
+    """Convert the given snake case word into a camelCase one."""
+    parts = iter(snake_case_word.split("_"))
+    pascal = "".join(word.title() for word in parts)
```

Comment:
what about reusing the to_pascal_case function?

Reply:
Good call.